src/applications/vaos/services/appointment/transformers.js

```diff
@@ -381,6 +394,21 @@ function setContained(appt) {
 
       return null;
     }
+    case APPOINTMENT_TYPES.ccRequest: {
+      return appt.ccAppointmentRequest.preferredProviders.map(provider => {
+        return {
+          actor: {
+            name: provider.practiceName,
+            address: {
+              line: provider.address?.street,
```

Comment:
`line` should be an array, so `line: [provider.address.street]`

Also, I think we will always have an address, but if we don't, I'd want to leave out the `address` object altogether here.

Reply:
Done.